lib/routes/routes.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:flutter/material.dart';
+
+const String routeDashboard = '/dashboard';
```

Comment:
More of a grammatical thing but possibly dashboardRoute, myWorkoutsRoute and creatorsCornerRoute instead? Last two are a bit wordy but we can figure out alternatives later :)